test/acse-conf/acse-regression.json

```diff
@@ -36,16 +36,6 @@
       "location": "eastus"
     },
     {
-      "cluster_definition": "disks-managed/swarm-preAttachedDisks-vmas-windows.json",
```

Comment:
+1

we can remove managed disks for swarm, but we will still need a test to test swarm